### ISSUE TITLE
baksmali errors

### ISSUE THREAD

**Issue by yomon598** [JesusFreke/smali#479]
baksmali fails on disassembling the classes.dex found in the apk here https://apkpure.com/power-rangers-dash/com.Ironnos.PowerRangersDashSaban

**Comment by JesusFreke** [comment]
It looks like this apk has been generated by some form of obfuscator. It has a ton of nonsense classes/methods, which is all the errors that you see. If you watch logcat while installing the apk, you'll see that dex2oat generates a ton of verification errors as well.

I believe some obfuscators have a native library that will "fix up" the dex file in memory after it is loaded. I'm not sure if that's the case here or not. But I looked at some of the errors that baksmali produced for this apk, and they all look like valid errors.

In short, baksmali should disassemble the valid classes, and skip any of the nonsense classes.
